packages/scss/src/components/inputs/file/_file.scss

```diff
@@ -37,31 +38,36 @@
 
 .file {
 	&.mod-small {
-		padding: _theme("spacings.smaller") _theme("spacings.standard")
-		_theme("spacings.smaller") 5rem;
+		align-items: center;
+		display: flex;
+		justify-content: space-between;
```

Comment:
Perhaps a `flex-wrap: wrap;` could be added here for a better responsive behavior ?